src/employees.json

```diff
@@ -0,0 +1,450 @@
+[
```

Comment:
The build will fail for JSON files as the build tasks do not support them. Needs to be converted to TS.